extensions/blocks/recurring-payments/edit.jsx

```diff
@@ -56,6 +58,19 @@ class MembershipsButtonEdit extends Component {
 			editedProductRenewInterval: '1 month',
 		};
 		this.timeout = null;
+
+		// FIXME: In Jetpack, we cannot yet support the 'new', Upgrade Nudge-based
+		// flow, since the post-checkout redirect skips the Jetpack Onboarding Checklist in Calypso,
+		// where Akismet and VaultPress installation is triggered.
+		// Instead, we continue to show the legacy, in-block prompt to upgrade the site's plan.
+		// This however means that we have to display the block also if the site's on a Free plan.
+		//
+		// Check p7rd6c-23C-p2 for details and progress. If ready, remove the line below, and replace all
+		// checks for `this.hasUpgradeNudge` with `false`.
```

Comment:
Oops, I think I forgot to update that last sentence. I don't think that's accurate anymore :sweat_smile:

Reply:
Right, I think we rely on the nudge being enabled or disabled by the filter. We should be able to remove this entire comment.